File: Array&String/Sorting/CPP/mean_of_array_after_removing_some_elements.cpp
#include <algorithm>
#include <numeric>
#include <vector>
using namespace std;

class Solution {
    public:
        double trimMean(vector<int>& arr) {
            sort(arr.begin(), arr.end());
            double n = arr.size();
            double cut = n / 20;
            double sum = accumulate(arr.begin() + cut, arr.end() - cut, 0);
            return sum / (n - (2 * cut));
        }
    };

File: Array-String/Sorting/JS/mean_of_array_after_removing_some_elements.js
/**
 * @param {number[]} arr
 * @return {number}
 */
var trimMean = function(arr) {
    arr.sort(function(a, b){return a - b});
    const n = arr.length;
    const cut = n / 20;
    const sum = arr.slice(cut, -cut).reduce((partSum, a) => partSum + a, 0);
    return sum / (n - (2 * cut));
};